Flaky Test: Secrets page base functionality

The test `Secrets page base functionality` is flaky.

```
     AssertionError: Timed out retrying after 10000ms: Expected to find element: `[data-testid="secret-card--toEverybody"]`, but never found it.
```

Example:
- https://app.circleci.com/pipelines/github/influxdata/monitor-ci/3965/workflows/a10a4a2c-9297-4f4b-8034-e31c4dbaedf8/jobs/68858

--- a/cypress/e2e/shared/secrets.test.ts
+++ b/cypress/e2e/shared/secrets.test.ts
@@ -27,7 +27,8 @@ describe('Secrets', () => {
     // Create secrets via the API, check visibility and sorting
     cy.get('@org').then(({id: orgID}: Organization) => {
       cy.upsertSecret(orgID, {toEverybody: 'rupees baby'})
-        .then(() => {
+        .then((resp: Response) => {
+          expect(resp.status).to.eq(204)
           cy.reload()
           cy.getByTestID('secret-card--toEverybody').should('be.visible')
           cy.getByTestID('copy-to-clipboard--toEverybody').should('exist')

